[Editorial] Update with review tweaks (#737)
w3c/wcag2ict: introduction.md

[Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/2013/NOTE-wcag2ict-20130905/), approved in September 2013, described how WCAG 2.0 could be applied to non-web documents and software. WCAG2ICT was organized to mirror WCAG's sections: Perceivable, Operable, Understandable, and Robust.  WCAG2ICT clarified when and how WCAG success criteria could be applied to non-web documents and software.  Some were applicable without modification and some were applicable with edits and/or notes.  Glossary terms were also reviewed. Level AAA success criteria were not addressed in the 2013 WCAG2ICT Working Group Note. 

> The 2013 version of WCAG2ICT has been relied upon in regulations and legislation.  One example is standards that reference or incorporate WCAG2ICT, such as \[\[etsi-en-301-549\]\] in Europe, and others in India, Kenya, and Australia.  Another example is Section 508 [Application of WCAG 2.0 to Non-Web ICT](https://www.federalregister.gov/documents/2017/01/18/2017-00395/information-and-communication-technology-ict-standards-and-guidelines#h-36) in the United States, which looked to WCAG2ICT for detailed direction on providing specific guidance for how to apply WCAG to non-web technology, as well as listing exceptions where WCAG success criteria do not apply.  Section 508 incorporated by reference WCAG as the [Accessibility Standard applicable to non-web documents](https://www.access-board.gov/ict/#E205.4) and requires [WCAG Conformance for non-web software](https://www.access-board.gov/ict/#E207.2).

Reviewer: The edits to this paragraph changes the meaning. the India, Kenya and Australia are country-specific adoptions of the EN 301 549. Therefore, reverting that sentence back to the original. Also, Section 508 was using the WCAG2ICT guidance, not providing its own, which led to exceptions where certain success criteria were not applied.
```suggestion
The 2013 WCAG2ICT has been relied upon in regulations and legislation. One example is \[\[etsi-en-301-549\]\] (Europe) and other standards that reference or incorporate EN 301 549 (e.g., India, Kenya, Australia).  Another example is Section 508 [Application of WCAG 2.0 to Non-Web ICT](https://www.federalregister.gov/documents/2017/01/18/2017-00395/information-and-communication-technology-ict-standards-and-guidelines#h-36) in the United States, which looked to WCAG2ICT's guidance for how to apply WCAG to non-web technology which led to a few  exceptions where WCAG success criteria are not applied by Section 508 to non-web technology.  Section 508 incorporated by reference WCAG as the [Accessibility Standard applicable to non-web documents](https://www.access-board.gov/ict/#E205.4) and requires [WCAG Conformance for non-web software](https://www.access-board.gov/ict/#E207.2).
```

Author: > The edits to this paragraph changes the meaning. the India, Kenya and Australia are country-specific adoptions of the EN 301 549. Therefore, reverting that sentence back to the original. Also, Section 508 was using the WCAG2ICT guidance, not providing its own, which led to exceptions where certain success criteria were not applied.

Sorry for that @maryjom and thanks for catching this. In order to make this paragraph easier to parse (a few things are still not clear IMO), would the following work for line 8 without changing the intended meaning?:

WCAG2ICT 2013 has been relied upon in regulations and legislation. One example is the European standard, \[\[etsi-en-301-549\]\], which relies upon WCAG2ICT, along with the standards that reference or incorporate EN 301 549 (e.g., India, Kenya, Australia).  Another example is Section 508 [Application of WCAG 2.0 to Non-Web ICT](https://www.federalregister.gov/documents/2017/01/18/2017-00395/information-and-communication-technology-ict-standards-and-guidelines#h-36) in the United States, which uses WCAG2ICT to provide guidance on how to apply WCAG to non-web technology, resulting in a few exceptions where WCAG success criteria are not applied by Section 508 to non-web technology.  This is important because Section 508 incorporates by reference WCAG as the [Accessibility Standard applicable to non-web documents](https://www.access-board.gov/ict/#E205.4) and requires [WCAG Conformance for non-web software](https://www.access-board.gov/ict/#E207.2).

Reviewer: Again, this changes the meaning of the original text. It flips the notion that certain criteria have exceptions from the sentences about the EN 301 549 to instead be on 508. The original text was imperfect as well. 

My take on an update:
```suggestion
The 2013 version of WCAG2ICT has been relied upon in regulations and legislation. An example is the \[\[etsi-en-301-549\]\] (Europe) as well as other standards that reference or incorporate EN 301 549 (e.g., India, Kenya, Australia). Another example is the U.S. Section 508’s [Application of WCAG 2.0 to Non-Web ICT](https://www.federalregister.gov/documents/2017/01/18/2017-00395/information-and-communication-technology-ict-standards-and-guidelines#h-36) where WCAG was incorporated by reference into Section 508 as the [accessibility standard applicable to non-web documents](https://www.access-board.gov/ict/#E205.4) and also requires [WCAG conformance for non-web software](https://www.access-board.gov/ict/#E207.2).

These standards looked to WCAG2ICT for detailed direction and guidance on how to apply WCAG non-web technology. The WCAG2ICT guidance also led to a few exceptions where specific success criteria are not required in non-web contexts.
```

Reviewer 2: @tamsinewing555 and @maryjom , would it be worthwhile to say Section 508 of the Rehabilitation Act vs. the U.S. Section 508’s if we are editing phrasing? That may be helpful for those not specifically in the accessibility space and where section 508 fits in to the US Fed level.

Reviewer: @ChrisLoiselle Not sure it's needed. It does get confusing, but we're really meaning the regulation, not the law. Since links are provided, I'd prefer to leave it as stated.

@tamsinewing555 I have incorporated your two edits into my suggested changes above.

Reviewer: @tamsinewing555 Can I merge in the rest of the changes as they are being proposed in the suggestions? I'm trying to get a full WCAG2ICT document together for the AG WG to review and publish ASAP.

Although this document covers a wide range of issues, it is not able to address all the needs of all people with disabilities. Since WCAG 2 was developed for the Web, addressing accessibility for non-web documents and software may involve requirements and considerations beyond those included in this document. Authors and developers are encouraged to seek relevant advice about current best practices to ensure that non-web documents and software are as accessible as possible to people with disabilities. The following supporting documents contain helpful information for learning about the user needs, intent, and generalized implementation techniques to support a wider range of people with disabilities:

* [WCAG 2 Overview](https://www.w3.org/WAI/standards-guidelines/wcag/)
* [Techniques for WCAG 2.2](https://www.w3.org/WAI/WCAG22/Techniques/) \[\[WCAG22-TECHS\]\]
* [How to Meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG22/quickref/)
* [Additional Accessibility Guidelines Working Group - Publications](https://www.w3.org/groups/wg/ag/publications/)
  

> While WCAG 2 was designed to be technology neutral, it assumes the presence of a “user agent” such as a browser, media player, or assistive technology that’s used as a means to access web content. As a result, the application of WCAG 2 to documents and software in non-web contexts necessitates some interpretation in order to determine how the intent of each WCAG 2 success criterion could be met in these different contexts of use. Therefore, the bulk of the WCAG2ICT Task Force's work involved evaluating how each WCAG 2 success criterion would apply to the context of non-web ICT.

Reviewer: Avoiding contractions.
```suggestion
While WCAG 2 was designed to be technology neutral, it assumes the presence of a “user agent” such as a browser, media player, or assistive technology that is used as a means to access web content. As a result, the application of WCAG 2 to documents and software in non-web contexts necessitates some interpretation in order to determine how the intent of each WCAG 2 success criterion could be met in these different contexts of use. Therefore, the bulk of the WCAG2ICT Task Force's work involved evaluating how each WCAG 2 success criterion would apply in the context of non-web ICT.
```

The glossary terms in WCAG 2 were also reviewed and most of them applied to non-web documents and software, as written. Some applied with additional notes or edits (largely related to phrases like “web page(s)”), and a small number of terms were only used in Level AAA success criteria, which are not addressed by the WCAG2ICT Note at this time.

### Excluded from Scope

The following are out of scope for this document:

*   This document does not seek to determine which WCAG 2 provisions (principles, guidelines, or success criteria) should or should not apply to non-web documents and software, but rather &mdash; if applied &mdash; *how* they would apply.

> *   This document does not propose changes to WCAG 2 or its supporting documents; and it does not include interpretations for implementing WCAG 2 in web technologies &mdash; note that during the development of this document, the WCAG2ICT Task Force did seek clarification on the intent of a number of the success criteria, which led to clarifications in the Understanding WCAG 2 document.

Reviewer: IMO, it's ok for a bullet to have multiple sentences.
```suggestion
*   This document does not propose changes to WCAG 2 or its supporting documents. It also does not include interpretations for implementing WCAG 2 in web technologies. Note that during the development of this document, the WCAG2ICT Task Force did seek clarification on the intent of a number of the success criteria, which led to clarifications in the Understanding WCAG 2 document.
```

Author: > IMO, it's ok for a bullet to have multiple sentences.

@maryjom - I'm happy to accept these reversions to remove the em dashes because we don't currently have a single WAI style guide to point to. Note that we are working on this, including guidance for multi-sentence list items :) 

The proposal we're discussing in this instance is to keep individual list items to no more than one complete sentence as a best practice for enhancing readability on the web. If a subsequent clause clarifies something in the previous clause in the list item, this could be signalled using an em dash, which sets off extra information relating to the previous clause and keeps a close connection with it, semantically.

Here is the rationale, along with some considerations for the list we're discussing in this issue: 

While multiple sentences are acceptable, if the bullet points are excessively long, this can defeat the purpose of using a list for quick comprehension. 

The list design needs to maintain consistency and clarity, and to build rhythm for readability. Multiple sentences in a list are acceptable if they enhance clarity — as long as all the list items conform to a similar pattern.

In other words, if one bullet point contains multiple sentences, all the other bullet points in the same list should also ideally contain multiple sentences, or at least be similar in structure. 

In the case of this list, it's hard to create consistency and build rhythm in the way it's currently designed because not all the list items are structured similarly (the second and third list items are especially problematic and IMHO do not help readers to easily and quickly parse the information). 

Furthermore. in the case of line 46, this list item has two topics: (a) no changes are proposed; and b) no interpretations for implementing WCAG 2 in web technologies are included). If you separate these topics into two complete sentences within the list item, then I propose it might be better to make the second sentence a separate list item, together with the 'note', which does not appear to be related to the first sentence — or at least, this is not clear to me (but perhaps I misunderstand it?).

Reviewer: I've edited my suggestion above to split line 46 into two bullets. Only some of the bullets required further explanation and though the rhythm may seem off by doing so, I don't want to add an extra explanation on bullets that don't have it when it isn't needed.

Reviewer 2: I would like to second @maryjom 's comment on over / extra explanation.

Author: +1 to splitting line 46 into two bullets and to not adding explanation when it is not needed.

> *   This document is not sufficient by itself to ensure accessibility in non-web documents and software &mdash; this is because, as a web standard, WCAG does not fully cover all accessibility requirements for non-user interface aspects of platforms, user-interface components as individual items, or ICT with closed functionality (where there is no assistive technology to communicate programmatic information).

Reviewer 3: Line 48 doesn't quite complete the explanation as to why "this document" (wcag2ict) is not sufficient by itself.

Current:
> This document is not sufficient by itself to ensure accessibility in non-web documents and software. This is because, as a web standard, WCAG does not fully cover all accessibility requirements for non-user interface aspects of platforms, user-interface components as individual items, or ICT with closed functionality (where there is no assistive technology to communicate programmatic information).

Proposed:
> This document is not sufficient by itself to ensure accessibility in non-web documents and software. This is because this document does not address accessibility requirements beyond those covered by WCAG which, as a web standard, does not fully cover all accessibility requirements for non-user interface aspects of platforms, user-interface components as individual items, or ICT with closed functionality (where there is no assistive technology to communicate programmatic information).

        * [style property](#dfn-style-properties)
        * [target](#dfn-targets)
        * [up event](#dfn-up-event)
* Updated terms:
    * [large scale](#dfn-large-scale)
    * [set of web pages](#dfn-set-of-web-pages)
    * [set of non-web documents](#set-of-documents)
    * [set of software programs](#set-of-software-programs)
* Updated sections:
    <div class="note">
      <ul>

>         <li>Most of the existing sections have undergone WCAG2ICT Task Force review and updates.</li>

Reviewer: Updated the text for accuracy. It was not quite right even before the suggested edits.
```suggestion
        <li>The existing sections have all undergone WCAG2ICT Task Force review to identify updates needed.</li>
```

>         <li>Most of the existing sections have undergone WCAG2ICT Task Force review and updates.</li>

Reviewer: ```suggestion
        <li>All of the existing sections have undergone WCAG2ICT Task Force review to identify the necessary updates.</li>
```
